docs/CONFIG.md

```diff
@@ -26,6 +26,21 @@ are based.
 The following additional steps are required:
 - You need to give cloud-api admin user permission to create and modify DNS Views. Instructions on how to add permission to "cloud-api-only" group is included in the video. Follow the same instructions to add "All DNS Views" permission under the "DNS Permissions" Permssion Type.
 
+Configuring Cloud Extensible Attributes using create-ea-defs tool
+-----------------------------------------------------------------
+If the "Cloud Network Automation" license is activated, then the Cloud Extensible Attributes used by the docker-ipam-plugin
+can be defined using the create-ea-defs tool in the infoblox/docker-ipam-tools docker image.
+
+To run create-ea-defs:
+```
+docker run infoblox/docker-ipam-tools create-ea-defs --debug --grid-host 10.120.21.150 --wapi-username=admin --wapi-password=infoblox --wapi-version=2.3
+```
+
+To use the configuration file for create-ea-defs:
+```
+docker run -v /etc/infoblox:/etc/infoblox infoblox/docker-ipam-tools create-ea-defs --debug --conf-file docker-infoblox.conf
+```
+
 Manual Configuration of Cloud Extensible Attributes
```

Comment:
A few comments:

- I think the list below is incomplete. For example, "Network Name" EA is missing.
- Might want to consider removing this section, since this can be done using the create-ea-defs tool described above.
- Lastly, if we choose to keep this section, we want to make it clear this is an alternative to using the create-ea-defs tool.